perf/load/pilot/README.md

```diff
@@ -2,23 +2,49 @@
 
 ## Basic Setup
 
-This folder includes a chart for light weight pilot performance measurements. Each workload only has one Envoy sidecar, single
-container in a Pod.
+This folder includes a chart for light weight pilot performance measurements. Each workload only has
+one Envoy sidecar, single container in a Pod.
 
 In 1.1.2 release, as resource consumption data point,
 Envoy sidecar consumes 5m CPU and 128M memory.
 
 ## Config Push Latency Test
 
+Config push measures the latency of the CDS push, the time starting from service discovery
+changes to the corresponding CDS changes reflected in the Envoy config.
+
+Several special installation setup is needed.
+
+- Remove Prometheus metrics drop in the config map
+- Ensure the cluster is appropriated provisioned with enough memory
+
 `./load_test.py`, starts large deployment, measure how long it takes for all
 sidecar to receive the same cds version.
 
-TODO
-- ensure prometheus scrape interval is set correctly
-- `envoy_cluster_manager_cds_version` does not work well when sidecar resource is used.
-- Add annotation for emiting the specific clusters, "sidecar.istio.io/statsInclusionPrefixes": "TBD"
-- Using new proxy image: gcr.io/mixologist-142215/proxyv2:suffix4
-- Removing Promethues load report drop config
-  - `regex: '(outbound|inbound|prometheus_stats).*'`, 
-  - `regex: 'envoy_cluster_(lb|retry|bind|internal|max|original).*'`
-- Change to `envoy_cluster_manager_cds_version{namespace="pilot-load"}` focusing on the testing namespace.
\ No newline at end of file
+## Setup
+
+To support large performance testing, many things need to be specially tuned.
+
+- Install Istio version, e.g., 1.1.2.
+- Pilot resource consumption, 6GB memory, 5 replica.
+- Prometheus resource consumption, update to 0.5 cpu, 10GB memory.
+- Prometheus filtering.
+
+  ```yaml
+  source_labels: [ cluster_name ]
```

Comment:
why is this filtering needed?

Reply:
we need metrics `envoy_cluster_upstream_cx_total {cluster_name = 890 || outbound | svc-0.pilot-load.svc.cluster.local`

Reply:
however, we can retain everything by just remove this rule, I'll see prometheus having hard time to catch up in that case.